packages/teams-js/src/public/constants.ts

```diff
@@ -42,6 +42,11 @@ export enum HostName {
    * Teams
    */
   teams = 'Teams',
+
+  /**
+   * Teams 2.1
```

Comment:
This comment will get turned into user facing documentation. Are you okay with "Teams 2.1" being how users think of this?

Reply:
Thanks for pointing this out. Have changed the value to `Modern Teams`.